eventmesh-connectors/README.md

```diff
@@ -4,7 +4,7 @@
 A connector is a bridge that interacts with a specific external service or underlying data source (e.g., Databases) on behalf of user applications. A connector is either a Source or a Sink.
 
 ## Source
-A source connector obtains data from an underlying data producer and delivers it to targets, after original data has been transformed into CloudEvents. It doesn't limit the way how a source retrieves data. (e.g., A source may pull data from a message queue or act as an HTTP server waiting for data sent to it).
+A source connector obtains data from an underlying data producer, and delivers it to targets after original data has been transformed into CloudEvents. It doesn't limit the way how a source retrieves data. (e.g., A source may pull data from a message queue or act as an HTTP server waiting for data sent to it).
```

Comment:
I have also made modifications to the `eventmesh-connectors/README.md` file to correct the same type of error. The correct sequence of the data processing steps should be "obtains data from data producer" first, followed by "transforms it into CloudEvents," and finally "delivers it to targets". However, the original text conveys a misleading meaning, suggesting that the sequence is "transforms it into CloudEvents," then "obtains data from data producer" and "delivers it to targets."